hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/SqlConnector.java

```diff
@@ -310,22 +309,22 @@ default Vertex sink(
      * returned by {@link #getPrimaryKey(Table)}.
      */
     @Nonnull
-    default Vertex deleteProcessor(@Nonnull DAG dag, @Nonnull Table table) {
+    default Vertex delete(@Nonnull DAG dag, @Nonnull Table table) {
```

Comment:
I'd rather rename `sink` to `sinkProcessor`. `delete()` seems like the call should delete something.